styles/components/_tooltip.scss

```diff
@@ -0,0 +1,70 @@
+.tooltip {
```

Comment:
Modifiers should always have the `-` prefix (as in `-arrowleft`). In this case, it's better to have `-left` instead of `-arrow-left`.